WooCommerce/Classes/Tools/AggregateData/AggregateDataHelper.swift

```diff
@@ -0,0 +1,138 @@
+import Foundation
+import Yosemite
+
+
+// MARK: - A helper class for calculating aggregate data
+//
+final class AggregateDataHelper {
+    /// Calculate the total quantity of refunded products
+    ///
+    static func refundedProductsCount(from refunds: [Refund]) -> Decimal {
+        var refundedItems = [OrderItemRefund]()
+        for refund in refunds {
+            refundedItems.append(contentsOf: refund.items)
+        }
+
+        var quantities = [Decimal]()
+        for item in refundedItems {
+            quantities.append(item.quantity)
+        }
```

Comment:
nit: the above lines could be simplified into:

```
let refundedItems = refunds.flatMap({ $0.items })
let quantities = refundedItems.map({ $0.quantity })
```

Reply:
I'm leaving these changes for my next PR, otherwise I'll have some steep merge conflicts.